napari/_qt/layer_controls/widgets/_labels/qt_label_color.py

```diff
@@ -0,0 +1,187 @@
+                    ):
+                        painter.setPen(QColor(230, 230, 230))
+                        painter.setBrush(QColor(230, 230, 230))
+                    else:
+                        painter.setPen(QColor(25, 25, 25))
+                        painter.setBrush(QColor(25, 25, 25))
+                    painter.drawRect(i * 4, j * 4, 5, 5)
+        else:
+            color = np.round(255 * self._layer._selected_color).astype(int)
+            painter.setPen(QColor(*list(color)))
+            painter.setBrush(QColor(*list(color)))
+            painter.drawRect(0, 0, self._height, self._height)
+            self.color = tuple(color)
+
+    def disconnect_widget_controls(self) -> None:
+        disconnect_events(self._layer.events, self)
+
+    def deleteLater(self) -> None:
+        self.disconnect_widget_controls()
+        super().deleteLater()
+
+    def closeEvent(self, event) -> None:
+        """Disconnect events when widget is closing."""
+        self.disconnect_widget_controls()
+        super().closeEvent(event)
+
+
+class QtLabelControl(QtWidgetControlsBase):
+    """
+    Class that wraps the connection of events/signals between the current label
+    layer attribute and Qt widgets.
+
+    Parameters
+    ----------
+    parent: qtpy.QtWidgets.QWidget
+        An instance of QWidget that will be used as widgets parent
+    layer : napari.layers.Layer
+        An instance of a napari layer.
+
+    Attributes
+    ----------
+    labelColor : qtpy.QtWidget.QWidget
+        Wrapper widget for the selectionSpinBox and colorBox widgets.
+    selectionSpinBox : superqt.QLargeIntSpinBox
+        Widget to select a specific label by its index.
+        N.B. cannot represent labels > 2**53.
+    colorBox: QtColorBox
+        Widget that shows current layer label color.
+    labelColorLabel : napari._qt.layer_controls.widgets.qt_widget_controls_base.QtWrappedLabel
+        Label for the label chooser widget.
+    """
+
+    def __init__(
+        self, parent: QWidget, layer: Layer, tooltip: Optional[str] = None
+    ) -> None:
+        super().__init__(parent, layer)
+        # Setup layer
+        self._layer.events.selected_label.connect(
+            self._on_selected_label_change
+        )
+        self._layer.events.data.connect(self._on_data_change)
+
+        # Setup widgets
+        self.selectionSpinBox = QLargeIntSpinBox()
+        dtype_lims = get_dtype_limits(get_dtype(layer))
+        self.selectionSpinBox.setRange(*dtype_lims)
+        self.selectionSpinBox.setKeyboardTracking(False)
+        self.selectionSpinBox.valueChanged.connect(self.changeSelection)
+        self.selectionSpinBox.setAlignment(Qt.AlignmentFlag.AlignCenter)
+        self._on_selected_label_change()
+
+        self.labelColorLabel = QtWrappedLabel(trans._('label:'))
+        self.labelColor = QWidget()
+        self.labelColor.setProperty('emphasized', True)
+        color_layout = QHBoxLayout()
```

Comment:
```suggestion
        color_layout = QHBoxLayout()
        color_layout.setContentsMargins(0, 4, 0, 1)
```
Fixes layout issue I noticed.